Subject: foo
From: olh@suse.de
Patch-mainline: never

  CC [M]  arch/ppc/oprofile/common.o
  arch/ppc/oprofile/common.c:128: error: conflicting types for `oprofile_arch_init'
  include/linux/oprofile.h:51: error: previous declaration of `oprofile_arch_init'

--- ../linux-2.6.10-bk13/arch/ppc/oprofile/Kconfig	2004-12-24 22:34:26.000000000 +0100
+++ ./arch/ppc/oprofile/Kconfig	2005-01-10 18:23:50.257274109 +0100
@@ -1,6 +1,6 @@
 
 menu "Profiling support"
-	depends on EXPERIMENTAL
+	depends on EXPERIMENTAL && BROKEN
 
 config PROFILING
 	bool "Profiling support (EXPERIMENTAL)"
